$OpenBSD$

We lose out on openat, fstatat, etc., with _XOPEN_SOURCE 500 defined.

Index: tools/cpmemu/fileio.c
--- tools/cpmemu/fileio.c.orig
+++ tools/cpmemu/fileio.c
@@ -1,4 +1,3 @@
-#define _XOPEN_SOURCE 500
 #define _POSIX_C_SOURCE 200809
 #include <stdio.h>
 #include <stdlib.h>
@@ -437,4 +436,4 @@ int file_rename(cpm_filename_t* src, cpm_filename_t* d
 
 	int drivefd = get_drive_fd(src);
 	return renameat(drivefd, srcunixfilename, drivefd, destunixfilename);
-}
\ No newline at end of file
+}
